arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1229,14 +1257,14 @@
       }
     },
     "OperationsMonitoringProperties": {
-      "description": "The Operation Monitoring properties.",
+      "description": "The operations monitoring properties for the IoT hub. The possible keys to the dictionary are Connections, DeviceTelemetry, C2DCommands, DeviceIdentityOperations, FileUploadOperations.",
       "type": "object",
       "properties": {
         "events": {
           "type": "object",
           "additionalProperties": {
             "enum": [ "None", "Error", "Information" ],
```

Comment:
@rkmanda I think the correct definition for this ENUM should be `[ "None", "Error", "Error, Information" ]` as I have stated in my email to you. Below is part of the response of GET IoT Hub call after I have configured the operation monitoring properties to `Verbose` in Azure Portal. You can refer to the snapshot at the bottom.
```json
                "operationsMonitoringProperties": {
                    "events": {
                        "None": "None",
                        "Connections": "Error, Information",
                        "DeviceTelemetry": "Error, Information",
                        "C2DCommands": "Error, Information",
                        "DeviceIdentityOperations": "Error, Information",
                        "FileUploadOperations": "Error, Information"
                    }
                },
```

![image](https://cloud.githubusercontent.com/assets/2300784/20508681/397c38bc-b09e-11e6-9760-41d124f04796.png)